test/models/onnx/cum_sum_2d_dynamic_axis_input.prototxt

```diff
@@ -39,6 +39,8 @@ graph {
     type {
       tensor_type {
         elem_type: 6
+        shape {
+        }
```

Comment:
Is this change necessary?
We should probably treat missing shape as a dynamic one.